arbitrary-users-patch/check_sidecar_image_digests.sh

```diff
@@ -59,13 +67,17 @@ while read -r line; do
     echo "[INFO] Image ${base_image_name} has valid digest"
   fi
 done < "${SCRIPT_DIR}"/base_images
-mv "${SCRIPT_DIR}"/base_images  "${SCRIPT_DIR}"/base_images.copy
+mv "${SCRIPT_DIR}"/base_images.copy  "${SCRIPT_DIR}"/base_images
```

Comment:
I don't understand why you're copying `base_images` to `base_images.copy` then doing all your sed work inline, then copying the chages back to `base_images`.

Why not just `sed -i` in `base_images` without the copy and move?

Reply:
shellcheck complained at editing a file, while iterating over it https://github.com/koalaman/shellcheck/wiki/SC2094
so that was my solution 😃